doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -124,6 +129,12 @@ The “source of truth” for validation would need to be established such as wh
 
 If no runtime tests are specified, then the scorecard would only run the static tests and not depend on a running cluster.
 
+Allow tests to declare a set of labels that can be introspected by the scorecard at runtime.
```

Comment:
This might be obvious to everyone, and sorry if it is, but how does a test declare a set of labels?

Reply:
here is my current thinking and what I've been prototyping so far...to add labels to the CheckSpec Basic test, I add the labels to the 'TestInfo' struct for that test like so:

        return &CheckSpecTest{
                BasicTestConfig: conf,
                TestInfo: schelpers.TestInfo{
                        Name:        "Spec Block Exists",
                        Description: "Custom Resource has a Spec Block",
                        Cumulative:  false,
                        Labels:      labels.Set{"required": "true", "suite": "basic"},
                },
        }

Reply:
For now, they're statically defined in code. In the future, where we may support Kubernetes `Job`-based tests, they would be declared in `metadata.labels`